libriscan/biblios/templates/biblios/base.html

```diff
@@ -107,6 +107,65 @@
   
   <!-- Page-specific scripts -->
   {% block scripts %}{% endblock %}
+
+  <!-- Floating Help Button (Bug Report + Feature Request + Conditional Tutorials) -->
```

Comment:
Remove if this code is addressed in PR https://github.com/Crimson-Vision/Libriscan/pull/288/files